Dynatrace/dynatrace-operator
# Feature flag for configuring minimum interval between requests to Dynatrace API

# Description

Provide a feature flag to define the minimum interval (in minutes) between requests to the Dynatrace API. Defaults to 15 minutes.

## How can this be tested?
Deploy operator, chose a value for the feature flag and deploy a resp. configured dynakube. Observe operator logs, you should see log messages like this stating that requests to Dynatrace API have been skipped:
```
{"level":"info","ts":"2023-03-15T08:19:40.046Z","logger":"dynatrace-client","msg":"skipping token verification, last request was made less than 20 minutes ago, 17 minutes remaining until next request"}
{"level":"info","ts":"2023-03-15T08:19:40.188Z","logger":"dynakube-connectioninfo","msg":"skipping activegate connection info update, last request was made less than 20 minutes ago, 17 minutes remaining until next request"}
{"level":"info","ts":"2023-03-15T08:19:40.188Z","logger":"dynakube-connectioninfo","msg":"skipping oneagent connection info update, last request was made less than 20 minutes ago, 17 minutes remaining until next request"}
```
Your configured limit should be reflected in the log lines, the remaining time should decrease.

## Checklist
